### DISCUSSION TITLE
萨米肉鸽源石锭投资问题

### DISCUSSION THREAD

**Discussion by MaYi0395** [MaaAssistantArknights/MaaAssistantArknights#8279] [Q&A]
![微信图片_20240207175442](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/158799147/c465712a-d0cb-415e-ad73-e6d8ad44515b)
萨米肉鸽投资一直异常，无法正常刷投资，但是水月和傀影肉鸽都正常

**Comment by status102** [comment]
提交log发issue，没log看不出问题
